tests/reg_tests/refs/adjoint_euler_scalar_jst_tut_wing.json

```diff
@@ -1,86 +1,87 @@
 {
+    "Adjoint States": 0.0,
```

Comment:
This entry should be the only one changing in the `.json` files. There is no point changing the reference values for this PR, especially if the training is done on a local machine.